data/h3_data_importer/delete_h3_tables.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import click
+import psycopg
+
+from utils import get_connection_info
+
+log = logging.getLogger("delete_h3_tables")
+logging.basicConfig(level=logging.INFO)
+
+
+@click.command()
+@click.option("--drop-contextuals", is_flag=True)
+@click.option("--dry-run", is_flag=True)
+def main(drop_contextuals: bool, dry_run: bool):
+    with psycopg.connect(get_connection_info()) as conn:
+        with conn.cursor() as cursor:
+            # find all the tables that start with h3_grid*
+            cursor.execute(
+                """SELECT table_name
+            FROM information_schema.tables as tables
+            LEFT JOIN  h3_data on h3_data."h3tableName" = tables.table_name
+            WHERE tables.table_name LIKE 'h3_grid%' and
+            h3_data."h3tableName" is null;
+            """
+            )
+            tables_to_drop = cursor.fetchall()
+            if tables_to_drop:
+                if not dry_run:
```

Comment:
Not sure why we need this

Reply:
I'm not sure either xd. Since it is deleting data I wanted to be sure that it is dropping what I want before running this for the real but since this will be mostly called from inside the system it does not make much sense really. I does not hurt tho. I don't know, up to you!